build.rs

```diff
@@ -77,7 +77,8 @@ fn write_cpp_conf_pri(path: &Path) -> Result<(), Box<dyn Error>> {
     let mut f = fs::File::create(path)?;
 
     writeln!(&mut f)?;
-    writeln!(&mut f, "INCLUDEPATH = /usr/local/include")?;
+    #[cfg(target_os = "macos")]
+    writeln!(&mut f, "INCLUDEPATH += {}", get_boost_path()?)?;
```

Comment:
Instead of making this mac-only, I think what you want is to only add the path if it isn't /usr/include?